src/stash/src/lib.rs

```diff
@@ -920,3 +925,38 @@ where
         deleted
     }
 }
+
+fn consolidate_updates<I>(rows: I) -> impl Iterator<Item = ((Value, Value), Timestamp, Diff)>
```

Comment:
Why do we need to serialize the keys and values to consolidate them?

Reply:
Because `serde_json::Value` doesn't implement `Ord`. Added a comment.